index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio - Microverse</title>
+  </head>
+  <body>
+    <header id="header">
+      <div class="container">
+        <div class="app-bar">
+          <img src="images/App bar.png" alt="mobile status-bar" />
+        </div>
+        <!-- navigation starts-->
+        <nav>
+          <a href="#" id="logo"><h2>MOZO</h2></a>
+          <div class="Menu-bar">
+            <a href="#"
+              ><img src="images/menu-bar.png" class="menu-bar" alt="menu-bar"
+            /></a>

```

Comment:
[OPTIONAL]- For improved user experience, I will suggest that when users click on your logo, it navigates to the index page. You can add an actual link to your `a` tag 👍 👍